.clang-format

```diff
@@ -3,4 +3,7 @@ Language: Cpp
 BasedOnStyle:  LLVM
 ColumnLimit: 120
 IndentWidth: 4
-AccessModifierOffset: -4
\ No newline at end of file
+AccessModifierOffset: -4
+PointerAlignment: Middle
+AllowShortBlocksOnASingleLine: Always
```

Comment:
I don't have a strong opinion.  Probably it's a good idea to fix it to something.

As to my not-so-strong opinion, `Never` would mean that an empty block needs to be written as
```
while (true) {
}
```
which looks worse than
```
while (true) {}
```
so `Always` sounds reasonable.  It looks like there's also `Empty`, but one could argue against this since it adds a special case.

Reply:
OK, let's go with `Always` for now. We are still searching for the best configuration, so it's flexible for now.